lib/setupmanagers/qemuhck/devices/iommu.rb

```diff
@@ -0,0 +1,29 @@
+# typed: true
+# frozen_string_literal: true
+
+require 'sys/cpu'
+
+module AutoHCK
+  class IommuDevice
+    extend T::Sig
+
+    def initialize(logger)
+      @logger = logger
+    end
+
+    sig { returns(Models::QemuHCKDevice) }
+    def qemu_hck_device
+      vendor = Sys::CPU.processors[0].vendor_id
```

Comment:
The gem is too complicated just to get the vendor ID. You can do: `File.read('/proc/cpuinfo').match(/^vendor_id\s*:\s*(.*)/)[1]`